src/global.ts

```diff
@@ -0,0 +1,12 @@
+declare global{
+    var activeUserId : string
+}
+globalThis.activeUserId = ""
```

Comment:
Not an essential change but we can use `null` instead of an empty string


```suggestion
declare global{
    var activeUserId : string | null
}
globalThis.activeUserId = null
```